src/components/ReportUrlFetcher.vue

```diff
@@ -123,18 +123,16 @@ export default class ReportUrlFetcher extends ReportUrlFetcherProps {
       try {
         const response = await fetch(this.url, { headers })
         this.state = "ready"
-        const contentType = response.headers.get("content-type")
-        if (
-          response.ok &&
-          contentType &&
-          contentType.indexOf("application/json") !== -1
-        ) {
+        if (response.ok) {
           const report = await response.json()
+          if (!report) {
+            throw new Error("Response from GitLab doesn't look like JSON")
```

Comment:
```suggestion
            throw new Error("Response from URL doesn't look like JSON")
```

Reply:
Fetching from a URL does not have to be from GitLab, so let's keep the error message generic.